include/nbl/ext/CentralLimitBoxBlur/CBlurPerformer.h

```diff
@@ -0,0 +1,130 @@
+// Copyright (C) 2018-2020 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#ifndef _NBL_EXT_BLUR_C_BLUR_PERFORMER_INCLUDED_
+#define _NBL_EXT_BLUR_C_BLUR_PERFORMER_INCLUDED_
+
+#include "nabla.h"
+
+namespace nbl
+{
+namespace ext
+{
+namespace CentralLimitBoxBlur
+{
+
+struct uvec4
+{
+    uint32_t x, y, z, w;
+};
+#include "nbl/builtin/glsl/ext/CentralLimitBoxBlur/parameters_struct.glsl"
+
+class CBlurPerformer final : public core::IReferenceCounted
+{
+public:
+    _NBL_STATIC_INLINE_CONSTEXPR uint32_t DefaultWorkgroupSize = 256u;
+    _NBL_STATIC_INLINE_CONSTEXPR uint32_t PassesPerAxis = 3u;
```

Comment:
few syntax pointers:
- use nested namespace, like this `namespace nbl::ext::CentralLimitBoxBlur`
- you can write `static inline constexpr` now, no need for the `_NBL_STATIC_INLINE_CONSTEXPR`